dephell/converters/sdist.py

```diff
@@ -183,7 +184,16 @@ def _write_additional_files(self, *, tar, project, subdir):
                     name=str(tests_path),
                     arcname=subdir + tests_path.name,
                     filter=self._set_uid_gid,
+                    recursive=False,
```

Comment:
Instead of disabling recursion and manually adding each file after, I would recommend using an expanded filter function. `_set_uid_gid()` will clean the mode and uid/gid of the files, but if it also returned None for paths that contain __pycache__ then those files will be dropped from the archive.

Reply:
Thank you for your feedback. I'll try to implement a filter function.

Reply:
This approach was much better than mine. Thanks a lot!

Reply:
Great!